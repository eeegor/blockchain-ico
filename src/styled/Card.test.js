import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Card } from '.';

describe('Card', () => {
	test('renders card', () => {
		const tree = renderer.create(<Card>Hello</Card>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders card with no background', () => {
		const tree = renderer.create(<Card noBg>Hello</Card>).toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('background', 'none !important');
	});
});
