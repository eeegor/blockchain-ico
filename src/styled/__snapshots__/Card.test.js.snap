// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders card 1`] = `
.c0 {
  padding: 12px;
  width: 100%;
  overflow: hidden;
  overflow-x: scroll;
  margin-left: auto;
  margin-right: auto;
  height: 100%;
  border: 1px solid #fff2;
  border-radius: 4px;
  background: #fff;
  box-shadow: 2px 4px 5px rgba(0,14,70,.17);
}

<div
  className="c0"
>
  Hello
</div>
`;

exports[`Card renders card with no background 1`] = `
.c0 {
  padding: 12px;
  width: 100%;
  overflow: hidden;
  overflow-x: scroll;
  margin-left: auto;
  margin-right: auto;
  height: 100%;
  border: 1px solid #fff2;
  border-radius: 4px;
  background: #fff;
  box-shadow: 2px 4px 5px rgba(0,14,70,.17);
  background: none !important;
  border: none !important;
  box-shadow: none !important;
}

<div
  className="c0"
>
  Hello
</div>
`;
